bittensor/_metagraph/__init__.py

```diff
@@ -50,10 +51,15 @@ def __new__(
                     an entry point node from that network.
                 chain_endpoint (default=None, type=str)
                     The subtensor endpoint flag. If set, overrides the network argument.
+                _mock (:obj:`bool`, `optional`):
+                    For testing, if true the metagraph returns mocked outputs.
         """      
         if config == None: 
             config = metagraph.config()
         config = copy.deepcopy(config)
+        config.metagraph._mock = _mock if _mock != None else config.metagraph._mock
+        if config.metagraph._mock:
```

Comment:
Same concern again. If we are mocking, let's not put it in a config, this is for testing only. Mock object construction should happen in test code.